source/docs/software/advanced-control/trapezoidal-profiles.rst

```diff
@@ -0,0 +1,155 @@
+Trapezoidal Motion Profiles in WPILib
+=====================================
+
+.. todo:: link to conceptual motion profiling article
+
+.. note:: This article covers the in-code generation of trapezoidal motion profiles.  Documentation describing the involved concepts in more detail is forthcoming.
+
+.. note:: For a guide on implementing the ``TrapezoidProfile`` class in the :ref:`command-based framework <docs/software/commandbased/what-is-command-based:What Is "Command-Based" Programming?>` framework, see :ref:`docs/software/commandbased/profile-subsystems-commands:Motion Profiling through TrapezoidProfileSubsystems and TrapezoidProfileCommands`.
+
+.. note:: The ``TrapezoidProfile`` class, used on its own, is most useful when composed with a custom controller.  To integrate it with a WPILib ``PIDController``, see :doc:`profiled-pidcontroller`.
```

Comment:
```suggestion
.. note:: The ``TrapezoidProfile`` class, used on its own, is most useful when composed with a smart motor controller with a built-in controller.  To integrate it with a WPILib ``PIDController``, see :doc:`profiled-pidcontroller`.
```

Reply:
I clarified this with slightly different wording, since that's not the only sort of custom controller one might want to use.